wiki/Beatmap/Difficulty/id.md

```diff
@@ -5,25 +5,24 @@ tags:
 # Tingkat kesulitan
 
-*Artikel ini tidak membahas seputar [Beatmap](/wiki/Beatmap) secara umum.*
+*Berbeda dengan [Beatmap](/wiki/Beatmap).*
 
-Istilah **tingkat kesulitan beatmap** (*beatmap difficulty*) secara garis besar merujuk pada berkas-berkas [dengan ekstensi `.osu`](/wiki/osu!_File_Formats/Osu_(file_format)) yang tersimpan di dalam folder masing-masing beatmap. Berkas-berkas ini merupakan bentuk manifestasi dari level-level permainan yang ada dan berisikan antara lain informasi seputar [hit object](/wiki/Hit_object) pada [mode permainan](/wiki/Game_mode) yang terkait serta [timing](/wiki/Beatmapping/Timing_section) dari lagu yang dipergunakan. Pada umumnya, seberapa sukar suatu tingkat kesulitan dapat dilihat dari nama dan [star rating](/wiki/Beatmapping/Star_rating) dari tingkat kesulitan tersebut.
+Sebuah **tingkat kesulitan beatmap** adalah sebuah level permainan, yang dimana tersimpan secara teks file [dengan ekstensi `.osu`](/wiki/osu!_File_Formats/Osu_(file_format)). Terdiri dari [game mode](/wiki/Game_mode), yang diatur sesuai dengan [timing](/wiki/Beatmapping/Timing_section) Berkas-berkas ini merupakan bentuk manifestasi dari level-level permainan yang ada dan berisikan antara lain informasi seputar [hit object](/wiki/Hit_object) pada [mode permainan](/wiki/Game_mode) yang terkait serta [timing](/wiki/Beatmapping/Timing_section) yang disesuaikan dengan lagu spesifik. Tingkat kesulitan memiliki aspek yang berbeda, yang secara langsung mempengaruhi pengalaman pemain. Nama tingkat kesulitan dan [*star rating*](/wiki/Beatmapping/Star_rating) biasanya memberikan cukup informasi untuk menceritakan betapa menantangnya itu.
 
-Dalam ruang lingkup yang lebih spesifik, seberapa sukar suatu tingkat kesulitan turut dipengaruhi oleh hal-hal teknis seperti kompleksitas [pattern-pattern](/wiki/Beatmap/Pattern) yang ada dan [pengaturan tingkat kesulitan (*difficulty setting*)](/wiki/Beatmap_Editor/Song_Setup#difficulty) yang dipergunakan. Para pemain dapat bermain dengan memakai [modifier](/wiki/Game_modifier) untuk membuat suatu tingkat kesulitan menjadi lebih mudah atau lebih sukar untuk dimainkan.
+Hit object dari sebuah tingkat kesulitan menyusun sebuah [pola](/wiki/Beatmap/Pattern), yang juga bervariasi per mode permainan dan mengatur nada untuk tampilan dan nuansa keseluruhan tentang bagaimana kesulitan harus dimainkan. Permainan dari sebuah tingkat kesulitan beatmap juga dapat diubah oleh berbagai [pengaturan kesulitan](/wiki/Beatmap_Editor/Song_Setup#difficulty) yang diatur oleh pemilik beatmap atau dialihkan dengan [*game modifier*](/wiki/Game_modifier).
 
-Di samping itu, suatu berkas tingkat kesulitan juga mengandung berbagai data yang terkait dengan sampel-sampel [hitsound](/wiki/Beatmapping/Hitsound) yang dapat didengar di dalam permainan.
+Dari perspektif pendengar, sebuah tingkat kesulitan dapat menambah lagu dengan satu atau beberapa set sample audio, atau [*hitsound*](/wiki/Beatmapping/Hitsound) yang berfungsi sebagai masukan
```

Comment:
```suggestion
Dari perspektif pendengar, sebuah tingkat kesulitan dapat memperkuat lagu dengan menambahkan satu atau beberapa set sampel audio, atau [*hitsound*](/wiki/Beatmapping/Hitsound) yang berfungsi sebagai umpan balik saat sedang bermain.
```
ok well i don't know the best word to represent "augment" in indonesian, actually it's like "adding" or in indonesian is "menambahkan". imho it's really not correct and has a biased understanding. and so "memperkuat" is one of the words i got that can represent it.